mintapi/api.py

```diff
@@ -197,24 +197,29 @@ def set_user_property(self, name, value):
         if req_id not in response:
             raise Exception("Could not parse response to set_user_property")
 
-    def get_transactions_json(self, include_investment=False,
+    def get_transactions_json(self, start_date=None, include_investment=False,
```

Comment:
Let's make start_date the last argument, in case anyone is using this API elsewhere with positional arguments, so we don't break their code.